### PR TITLE
[Core]: Small Docstring Clarification for `BaseTool`

### PR DESCRIPTION
**Pull request by keenborder786**
- **Description:** `kwargs` are not being passed to `run` of the `BaseTool` which has been fixed
- **Issue:** #28114

### CODE REVIEW

File: libs/core/langchain_core/tools/base.py [hunk 1]
```diff
@@ -609,7 +609,7 @@ def run(
             run_id: The id of the run. Defaults to None.
             config: The configuration for the tool. Defaults to None.
             tool_call_id: The id of the tool call. Defaults to None.
-            kwargs: Additional arguments to pass to the tool
+            kwargs: Key Word Arugment to be passed to tool callbacks
```
Comment (kapicic) [inline libs/core/langchain_core/tools/base.py:612]:
```suggestion
            kwargs: Keyword arguments to be passed to tool callbacks
```

File: libs/core/langchain_core/tools/base.py [hunk 2]
```diff
@@ -721,7 +721,7 @@ async def arun(
             run_id: The id of the run. Defaults to None.
             config: The configuration for the tool. Defaults to None.
             tool_call_id: The id of the tool call. Defaults to None.
-            kwargs: Additional arguments to pass to the tool
+            kwargs: Key Word Arugmenst to be passed to tool callbacks
```
Comment (kapicic) [inline libs/core/langchain_core/tools/base.py:724]:
```suggestion
            kwargs: Keyword arguments to be passed to tool callbacks
```

**Comment by keenborder786** [comment]
@eyurtsev

**Comment by keenborder786** [comment]
@eyurtsev

**Review by efriis — APPROVED** [review]

**Pull request merged**